### PR TITLE
Add intense mode feature

### PR DESCRIPTION
**Pull request by Dielee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +39,8 @@ Create some settings file called `settings.local.json` (see [`settings.example.j
 
 `polling_schedule` sets the polling intervall in cron notation. For more Infomation have a look here: https://crontab.guru/
 
+`intense_fetch` sets the intense fetch intervall (seconds) and period of time (minutes)
```
Comment (MaxWinterstein) [inline README.md:42]:
```suggestion
`intense_fetch` sets the intense fetch interval (seconds) and period of time (minutes)
```

**Review by MaxWinterstein — COMMENTED** [review]
Love it.

Might drop some words about the intense_fetch functionality?

File: toogoodtogo_ha_mqtt_bridge/main.py [hunk 2]
```diff
@@ -395,6 +401,103 @@ def calc_timeout():
...
+            else:
+                logger.info("No running thread found. Doing nothing.")
+
+
+def register_fetch_sensor():
+    mqtt_client.publish(
+        f"homeassistant/switch/toogoodtogo_bridge/intense_fetch/config",
+        json.dumps(
+            {
+                "name": "TooGoodToGo - Intense fetch",
```
Comment (MaxWinterstein) [inline toogoodtogo_ha_mqtt_bridge/main.py:480]: I would propose to use just `Intense fetch` as name, it seeems to be trunkated

image

File: toogoodtogo_ha_mqtt_bridge/main.py [hunk 3]
```diff
@@ -395,6 +401,103 @@ def calc_timeout():
...
+                "icon": "mdi:history",
```
Comment (MaxWinterstein) [inline toogoodtogo_ha_mqtt_bridge/main.py:481]: Personal opinion:

maybe change the icon, i like `mdi:fast-forward`

image

**Comment by MaxWinterstein** [comment]
If i did not add the `intense_fetch` dict to my settings, the switch at home assistant feels somehow weird. I got a new button, but if I press it, nothing happens - beside the error log at the bridge itself.

I think we should either
a) hide the button when the setting is not set
b) move the setting to `settings.json` to have it pre-delivered for anyone who comes from an update, not a fresh installation

I like a) the most -- thinking of it as an advanced feature

File: toogoodtogo_ha_mqtt_bridge/main.py (hunk 2 above)
Comment (Dielee) [reply toogoodtogo_ha_mqtt_bridge/main.py]: Done with https://github.com/MaxWinterstein/toogoodtogo-ha-mqtt-bridge/pull/60/commits/4028f1fc4b968e439bb546cd541cf4dea18ae084

File: toogoodtogo_ha_mqtt_bridge/main.py (hunk 3 above)
Comment (Dielee) [reply toogoodtogo_ha_mqtt_bridge/main.py]: Done with https://github.com/MaxWinterstein/toogoodtogo-ha-mqtt-bridge/pull/60/commits/4028f1fc4b968e439bb546cd541cf4dea18ae084

**Comment by Dielee** [comment]
> If i did not add the `intense_fetch` dict to my settings, the switch at home assistant feels somehow weird. I got a new button, but if I press it, nothing happens - beside the error log at the bridge itself.
> 
> I think we should either a) hide the button when the setting is not set b) move the setting to `settings.json` to have it pre-delivered for anyone who comes from an update, not a fresh installation
> 
> I like a) the most -- thinking of it as an advanced feature

Yes, I like a the most, too. Good hint, will build this feature.

**Comment by Dielee** [comment]
Readme updated! Should all be finde now.

**Pull request merged**